requirements/docs.txt

```diff
@@ -8,5 +8,6 @@ sphinx-autobuild==0.7.1
 m2r==0.2.1
 mistune<2.0.0
 sphinx-notfound-page==0.6
+kolibri==0.15.11
```

Comment:
Please remove this - it pins our docs to a specific published version of Kolibri, rather than the current source branch.

I checked the readthedocs builds, and there have not been any failures from recent pushes.